dspace-server-webapp/src/main/java/org/dspace/app/rest/security/DSpaceRestPermission.java

```diff
@@ -18,7 +18,8 @@
     WRITE(Constants.WRITE),
     DELETE(Constants.DELETE),
     ADD(Constants.ADD),
-    ADMIN(Constants.ADMIN);
+    ADMIN(Constants.ADMIN),
+    STATUS(Constants.READ);
```

Comment:
I would prefer we add a comment here about the new `STATUS` type, as it's not described anywhere in this PR (I don't see any new code comments).  It could say something like `STATUS permissions allows someone to see the status of an object, without necessarily being able to fully READ it. For example, STATUS is used to allow withdrawn Items to be seen as withdrawn, while hiding all their metadata/links.`